src/CreateQuiz.js

```diff
@@ -31,11 +35,15 @@ class CreateQuiz extends React.Component {
         });
     }
 
+    deleteQuestion = (index) => {
+        let questions = this.state.questions;
+        questions.splice(index, 1);
+        this.setState({ questions: questions })
+    }
 
     addQuestion = () => {
         let questions = this.state.questions;
         questions.push({
```

Comment:
This might be working on accident, but this could be the source of some issues: if React doesn't see a _new_ `questions` array assigned, it doesn't know to rerender. Common patterns to fix this:

1. Concat without an argument copies an array: `let questions = this.state.questions.concat()`. Then you can `push()`.
2. Spreading into a new array: `let questions = [...this.state.questions]`. Then you can `push()`.
3. Spread into a new array and include the new element:
    ```js
    let questions = [
        ...this.state.questions,
        {
            question: '',
            // ...
    ```

The same concept applies to all your `questions` updates - updating needs to create a new array and swap in the update, delete needs to create a new array without the deleted index, etc. Google will find lots of "React state update array" reading.